src/content/docs/en/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,608 @@
+
+You will find [complete, step-by-step instructions for removing the old image integration](/en/guides/images/#remove-astrojsimage) in our Images guide.
+
+Migrating to `astro:assets` will also bring some new image options and features that you may now wish to use. Please see the full [v3.0 Image Upgrade Advice](/en/guides/images/#upgrade-to-v30-from-v2x) for full details!
+
+```js del={3,7}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import image from '@astrojs/image';
+
+export default defineConfig({
+  integrations: [
+    image(),
+  ]
+})
+```
+
+### Removed: `<Markdown />` component
+
+In Astro v1.x, Astro deprecated the `<Markdown />` component and moved it to an external package.
+
+Astro v3.0 completely removes the package `@astrojs/markdown-component`. Astro's `<Markdown />` component will no longer work in your project.
+
+#### What should I do?
+
+Remove all instances of the `@astrojs/markdown-component`.
+
+```astro del={2} title="src/components/MyAstroComponent.astro"
+---
+import Markdown from '@astrojs/markdown-component';
+---
+```
+
+To continue using a similar `<Markdown />` component in your code, consider using [community integrations](https://astro.build/integrations/) such as [`astro-remote`](https://github.com/natemoo-re/astro-remote). Be sure to update your `<Markdown />` component imports and attributes as necessary, according to the integration's own documentation. 
+
+Otherwise, delete all references to importing Astro's `<Markdown />` component and the component itself in your `.astro` files. You will need to rewrite your content as HTML directly or [import Markdown](/en/guides/markdown-content/#importing-markdown) from a `.md` file.
+
+### Removed: deprecated 1.x APIs
+
+In Astro v1.x, Astro deprecated config options as well as script/style `global` and `hoist` attributes types. These were kept for backwards compatibility.
+
+Astro v3.0 removes these deprecated APIs entirely. They can no longer be used in your Astro project.
+
+#### What should I do?
+
+If you are continuing to use v1.x APIs, use the new APIs for each feature instead:
+
+- Deprecated config options: See [the 0.26 migration guide](/en/guides/upgrade-to/v1/#new-configuration-api)
+- Deprecated script/style attribute types: See [the 0.26 migration guide](/en/guides/upgrade-to/v1/#new-default-script-behavior)
+
+### Removed: `image` from `astro:content` in content collections schema
+
+In Astro 2.x, the content collections API deprecated an `image` export from `astro:content` for use in your content collections schemas.
+
+Astro 3.0 removes this export entirely.
+
+#### What should I do?
+
+If you are using the deprecated `image()` from `astro:content`, remove it as this no longer exists. Validate images through [the `image` helper from `schema`](/en/guides/images/#update-content-collections-schemas) instead:
```

Comment:
Point out that this link isn't marked as broken, but this specific heading doesn't exist yet! till the v3 images page is merged

Reply:
Thanks, Yan! There will be some like this, since some headings on the new page are in fact the same as the ones on the existing page!